gym-service/src/test/features/trainer/trainer.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Trainer: Profile Management
+
+  Scenario: Successfully access profile
+    Given the trainer is registered with username: "testTrainer"
+    When the trainer requests the trainer profile
+    Then the response status for trainer should be 200
+    And the response should contain the trainer's profile information:
+      | firstName | lastName | specialization | traineeList | userIsActive |
+
+  Scenario: Access profile without JWT
+    Given the trainer is registered with username: "testTrainer"
+    When the trainer requests the trainer profile
+    Then the response status for trainer should be 403
```

Comment:
again, speaking of proper status codes missing authorization is 401. 403 when authorization provided, but not sufficient